scripts/build_layers.sh

```diff
@@ -10,7 +10,6 @@ set -e
 
 LAYER_DIR=".layers"
 LAYER_FILES_PREFIX="datadog_lambda_node"
-NODE_VERSIONS=("10.15" "12.13" "14.15")
```

Comment:
We run this locally while deploying, so I think we probably want to keep the ability to build all layers in a single command. We could have an env var or arg that limits the script to a single runtime. You can see an example of how we do that in the [./publish_layers.sh](https://github.com/DataDog/datadog-lambda-js/blob/main/scripts/publish_layers.sh#L28) script

Reply:
thanks, this should be resolved with 944c85e